### DISCUSSION TITLE
Tracking issue for BITCOIN: URI prefix in QR codes

### DISCUSSION THREAD

**Discussion by Kixunil** [btcpayserver/btcpayserver#2336] [Ideas / Feature Requests]
Not all wallets support upper-case `BITCOIN:` prefix despite it being standard. Using it would decrease the size of QR codes, so it's something that would be helpful but compatibility is more important.

This was already discussed in #2110 

This issue serves as tracking issue for wallet support.

Table of wallets supporting this:

Wallet Name | BITCOIN:BECH32 | BITCOIN:bech32 | bitcoin:BECH32 | bitcoin:bech32 | Validated By
-- | -- | -- | -- | -- | --
Atomic Wallet | ✔️ | ✔️ | ✔️ | ✔️ | @brijananand
BitBoxApp | ✔️ | ✔️ | ✔️ | ✔️ | @Stadicus
BitBoxApp for Android | ✔️ | ✔️ | ✔️ | ✔️ | @Stadicus
Bitcoin Wallet Android (Schildbach) | ✔️ | ✔️ | ✔️ | ✔️ | @freerko
Blixt Wallet | ✔️ | ✔️ | ✔️ | ✔️ | @hsjoberg
Blockchain Info | ❌ | ❌ | ❌ | ❌ | @pavlenex
Blockstream Green | ✔️ | ✔️ | ✔️ | ✔️ | @britttttk
BlueWallet | ✔️ | ✔️ | ✔️ | ✔️ | @britttttk
BLW Android | ✔️ | ✔️ | ✔️ | ✔️ | @anitaposch
BRD| ❌| ❌ | ✔️ | ✔️ | @pavlenex 
Breez | ✔️ | ✔️ | ✔️ | ✔️ | @Kixunil @brijananand
BTCPay Server | ✔️ | ✔️ | ✔️ | ✔️ | @pavlenex
Casa | ✔️ | ✔️ | ✔️ | ✔️ | @jlopp 
CashApp | ✔️ | ✔️ | ✔️ | ✔️ | @britttttk
ChivoWallet | ✔️ | ✔️ | ✔️ | ✔️ | @jquiros2
Coinbase Wallet | ❌ | ❌ | ✔️ | ✔️ | @Fichte42
Coinomi | ✔️ | ✔️ | ✔️ | ✔️ | @rockstardev 
Copay | ❌ | ❌ | ❌ | ❌ | @anitaposch
Eclair | ✔️ | ✔️ | ✔️ | ✔️ | @rockstardev
Edge | ✔️ | ✔️ | ✔️ | ✔️ | @pavlenex
Electrum 4.0.8 | ✔️| ✔️| ✔️ | ✔️ | @pavlenex 
Electrum Android | ❌ | ❌ | ✔️ | ✔️ | @LinoxBE @anitaposch
Exodus | ✔️ | ✔️ | ✔️ | ✔️ | @Fichte42 
Fully Noded | ❌ | ✔️ | ❌ | ✔️ | @pedromvpg
Hexa Wallet | ❌ | ❌ | ✔️ | ✔️ | @mutatrum
Ledger Live Desktop | ❌ | ✔️ | ❌ | ✔️ | @onyb
Ledger Live iOS | ❌ | ✔️ | ❌ | ✔️ | @woutersamaey
Lighting by L.Labs  | ❌ | ❌ | ❌ | ❌ | @brijananand
Moonshine | ✔️ | ✔️ | ✔️ | ✔️ | @pedromvpg
Muun | ✔️ | ✔️ | ✔️ | ✔️ | @Fichte42
Mycelium iOS | ❌ | ❌ | ❌ | ❌ | @britttttk
Mycelium 3.7.0.5 - Android | ✔️ | ✔️ | ✔️ | ✔️ | @vjsingh-lite
Phoenix | ✔️ | ✔️ | ✔️ | ✔️ | @Kixunil
Samourai | ❌ | ❌ | ✔️ | ✔️ | @Kixunil
Shakepay | ✔️ | ✔️ | ✔️ | ✔️ | @aiouy
Sparrow Wallet | ✔️ | ✔️ | ✔️ | ✔️ | @craigraw 
Specter Desktop | ❌ | ❌ | ❌ | ❌ | @mutatrum
Strike | ✔️ | ✔️ | ✔️ | ✔️ | @rockstardev
Trezor (wallet.trezor.io) | ❌ | ❌ | ✔️ | ✔️ |  @schulzemic 
Trezor Suite | ✔️ | ✔️ | ✔️ | ✔️ | @prusnak
Trust | ✔️ | ✔️ | ✔️ | ✔️ | @Fichte42
Wallet of Satoshi | ✔️ | ✔️ | ✔️ | ✔️ | @Kixunil
Wasabi | ✔️ | ✔️ | ✔️ | ✔️ | @johnsBeharry
Zap | ✔️ | ✔️ | ✔️ | ✔️ | @britttttk 
Zeus | ❌ | ❌ | ❌ | ✔️ | @davterra
Bitcoin.com (owned by Roger Ver, scammer) | ❌ | ❌ | ✔️ | ✔️ | @pavlenex

**Comment by Kixunil** [comment]
Damn, the data didn't copy and I really don't want to do it manually. Does anyone know some trick to do it automatically?

**Reply by Kukks** [reply]
fixed

**Comment by craigraw** [comment]
Can you (re) add:

Wallet | BITCOIN:BECH32 | BITCOIN:bech32 | bitcoin:BECH32 | bitcoin:bech32
-- | -- | -- | -- | --
Sparrow Wallet | Yes | Yes | Yes | Yes

Thanks

**Comment by hsjoberg** [comment]
Here's for [Blixt Wallet](https://github.com/hsjoberg/blixt-wallet)!


Wallet       | BITCOIN:BECH32 | BITCOIN:bech32 | bitcoin:BECH32 | bitcoin:bech32
------------ | -------------- | -------------- | -------------- | --------------
Blixt Wallet | ✔️             | ✔️              | ✔️             | ✔️

**Comment by pavlenex** [comment]
@Kixunil It seems we'll need a bit more initiative to bring this one back to life? Is there still interest to proceed with a tracker, and would you prefer if this is moved into an issue?

**Reply by Kixunil** [reply]
Good question. What would be the criteria to re-enable it? Which wallets need to support it?

**Reply by pavlenex** [reply]
I guess we can try to get as much as we can from that column, but we'd need to create a campaign once again to post issues on wallet's repos and see who's willing to implement it.
